ui/nuxeo-page-provider-display-behavior.js

```diff
@@ -272,6 +298,10 @@ export const PageProviderDisplayBehavior = [
             }
           }
         }
+        // if the view is not handling select all, we need to prevent deselect (for now)
```

Comment:
>(for now)

Why is it for now? Do we need to add more info to clarify when we won't be needing this anymore?

Reply:
When we are in select all mode, you cannot unselect anything, until we have the unselect some implemented in server side. Maybe I can clarify it in the comment.